url.go

```diff
@@ -20,8 +21,12 @@ func RedactURL(u *url.URL) string {
 		ru.User = url.UserPassword(ru.User.Username(), "redacted")
 	}
 	q := ru.Query()
-	if q.Get("sshkey") != "" {
-		q.Set("sshkey", "redacted")
+	if q.Has("sshkey") {
+		values := q["sshkey"]
+		for i := range values {
+			values[i] = "redacted"
+		}
+		q["sshkey"] = values
```

Comment:
Question: Is this reassignment required?

Reply:
Ah good catch, it's indeed not needed. Removed in [80a526c](https://github.com/hashicorp/go-getter/pull/528/commits/80a526cd1b22f0b770dfd158557341f890f3b4a7)